docs/getting_started.rst

```diff
@@ -65,30 +68,45 @@ Once all the necessary packages have been cloned, you need to build them. To do
     colcon build
 
 
-with the possible addition of flags depending on the specific use-case. Notice also that in most situations you will have to execute some additional commands before running the :code:`colcon build`, e. g. sourcing an external overlay,
-as is the case for ROS2 and ROS1 (see e. g. next section).
+with the possible addition of flags depending on the specific use-case. Notice also that in most situations you will
+have to execute some additional commands before running the :code:`colcon build`, e. g. sourcing an external overlay,
+as is the case for :code:`ROS2` and :code:`ROS1` (see e. g. next section).
 
 Once that's finished building, you can source the new colcon overlay:
 
 .. code-block:: bash
 
     source install/setup.bash
 
-The workspace is now prepared for running a :code:`soss` instance. From the fully overlaid shell, you will have to execute the :code:`soss` command
+The workspace is now prepared for running a :code:`soss` instance. From the fully overlaid shell, you will have to
+execute the :code:`soss` command
 followed by the name of the YAML configuration file that describes how messages should be passed
 among the middlewares involved.
 
 Once :code:`soss` is initiated, the user will be able to communicate the desired protocols.
 
-For more information on how to configure :code:`soss` via a YAML file, please refer to :ref:`Yaml Configuration <yaml configuration>`.
-For information on how to create your own custom **System-Handle**, see :ref:`System-Handle Creation <system-handle creation>` instead.
+For more information on how to configure :code:`soss` via a YAML file, please refer to :ref:`Yaml Configuration
+<yaml configuration>`. For information on how to create your own custom **System-Handle**, see
+:ref:`System-Handle Creation <system-handle creation>` instead.
+
+**Note**: the sourcing of the local colcon overlay is required every time the colcon workspace is opened in
+a new shell environment.
+As an alternative, you can copy the source command with the full path of your local installation to your 
```

Comment:
Avoid 2nd person?

Reply:
in this case it's ok, I would say, because you are deliberately and not accidentally addressing the reader.